public/app/features/explore/LogsContainer.tsx

```diff
@@ -20,8 +20,9 @@ import { getTimeZone } from '../profile/state/selectors';
 
 import { LiveLogsWithTheme } from './LiveLogs';
 import { Logs } from './Logs';
-import { addResultsToCache, clearCache, loadLogsVolumeData, setLogsVolumeEnabled } from './state/query';
+import { addResultsToCache, clearCache, loadSuppQueryData, setSuppQueryEnabled } from './state/query';
```

Comment:
> which is a object with key that is type of supp query

What is a "supp query"? I'd like to know if I'm missing context or maybe we should come up with a different name.

Reply:
Good point. I was trying to come up with good naming, but had hard time. So all suggestions are more than welcome! 🙂 First I was thinking supportingQuery, then supplementary, but these are quite long. But maybe better long than vague. I was also thinking additional, but it sounds vague as well. 🤔

Reply:
If `supporting` correctly describes the types of queries, I would just use the full word to avoid this ambiguity.

Reply:
They are not quite meta information, but what about `metaQueries`?

Reply:
I think it does, but I'd like to hear what other people think. Basically it is a query that is optionaly (at user's request) run in explore in addition to original query. Such as when you run log query and want to see log histogram. Or when you run metrics query and want to see log samples. I like the `supplementary` the most, but it is veeery long.

Reply:
As we use `meta` to name meta information from query, plus in Loki we use `metadataRequest` to get resources, I have decided to use `supplementary` to make it more distinguished in https://github.com/grafana/grafana/pull/60998/commits/a2940be22d2b828d555a81a37a30dcab12aa3732 and https://github.com/grafana/grafana/pull/60998/commits/162af478436d7a8d2232a0f9270ddb91f19f1f1e

Reply:
Oh, I did a mistake and renamed it to `supporting` instead of `supplementary`. Working on fix now.

Reply:
Fixed in https://github.com/grafana/grafana/pull/60998/commits/ccfb7a63a40d6b6649cb14463b07edc4f91b5226